spec.html

```diff
@@ -39767,6 +39952,27 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-typedarray-create-same-type" type="abstract operation">
+        <h1>
+          TypedArrayCreateSameType (
+            _exemplar_: a TypedArray,
+            _argumentList_: a List of ECMAScript language values,
+          ): either a normal completion containing a TypedArray or a throw completion
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It is used to specify the creation of a new TypedArray using a constructor function that is derived from _exemplar_. Unlike TypedArraySpeciesCreate, which can construct custom TypedArray subclasses through the use of @@species, this operation always uses one of the built-in TypedArray constructors.</dd>
+        </dl>
+        <emu-alg>
+          1. Assert: _exemplar_ is an Object that has [[TypedArrayName]] and [[ContentType]] internal slots.
```

Comment:
This assert isn't needed because _exemplar_ is given the type "a TypedArray" already in the header.

Reply:
https://github.com/tc39/ecma262/pull/2997#discussion_r1082149982